pkgs/applications/networking/browsers/firefox/wrapper.nix

```diff
@@ -164,7 +143,25 @@ let
       #                           #
       #############################
 
-    in stdenv.mkDerivation {
+      # TODO: remove this in the future
```

Comment:
when do we want to remove this actually? after branch off (of 21.03) or the release after that maybe?

Reply:
After the next release sounds reasonable.